src/lib/y2storage/planned/can_be_formatted.rb

```diff
@@ -58,14 +61,17 @@ def initialize_can_be_formatted
       #
       # Do nothing if #filesystem_type is not set.
       #
+      # FIXME: temporary API. It should be improved.
+      #
       # @param blk_dev [BlkDevice]
       #
       # @return [Filesystems::BlkFilesystem] filesystem
-      def create_filesystem(blk_dev)
+      def format!(blk_dev)
```

Comment:
please explain in comment why it have `!` in name. ( like that it modify device graph attached to argument ).